src/components/ClientForm/ClientForm.tsx

```diff
@@ -0,0 +1,165 @@
+"use client";
+
+import styles from "./ClientForm.module.scss";
+import { ClientFormDataDropdowns } from "../../assets/data/ClientFormData";
+import { Dropdown } from "../Dropdown/Dropdown";
+import { useState } from "react";
+import { sendContactForm } from "@/assets/apiLib/api";
+
+export function ClientForm() {
+  const [name, setName] = useState<string>("");
+  const [phone, setPhone] = useState<string>();
+  const [email, setEmail] = useState<string>();
+  const [trainingFor, setTrainingFor] = useState<string>(
+    "General Health and Fitness"
+  );
+  const [currentDaysWeek, setCurrentDaysWeek] = useState<string>("0");
+  const [futureDaysWeek, setFutureDaysWeek] = useState<string>("0");
+  const [currentFitness, setCurrentFitness] = useState<string>(
+    "1: relatively out of shape"
+  );
+  const [injuries, setInjuries] = useState<string>("No");
+  const [yesInjuries, setYesInjuries] = useState<string>();
+  const [favorite, setFavorite] = useState<string>("Cardio");
+  const [mostOften, setMostOften] = useState<string>("Cardio");
+  const [goals, setGoals] = useState<string>();
+  const [coachGoals, setCoachGoals] = useState<string>();
+
+  function handleChange(val: string, id: string) {
```

Comment:
all of these have an ID and a string value so you can replace all of this with

`
  setDropDownOptions(prev => {
    return {
      ...prev,
      [id]: val
    }
  })
`